backend/django/core/forms.py

```diff
@@ -293,7 +293,12 @@ def __init__(self, *args, **kwargs):
     absolute_max=10000,
 )
 LabelDescriptionFormSet = forms.inlineformset_factory(
-    Project, Label, form=LabelDescriptionForm, can_delete=False, extra=0
+    Project,
+    Label,
+    form=LabelDescriptionForm,
+    can_delete=False,
+    extra=0,
+    absolute_max=10000,
```

Comment:
Do we say anywhere that 10k is the max for labels? If it's not on the Label creation form we should add it.

Reply:
Good thinking, added.